smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/i18n/AMQPExceptions.java

```diff
@@ -0,0 +1,35 @@
+package io.smallrye.reactive.messaging.amqp.i18n;
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+/**
+ * Exceptions for AMQP Connector
+ * Assigned ID range is 16000-16099
+ */
+@MessageBundle(projectCode = "SRMSG")
```

Comment:
It might be a good idea to add length. Though it's the same as the default right now, if the default changed we would get different numbers output

Reply:
That would apply to all `@MessageBundle` usage

Reply:
What should the length be ? 6 ?

Reply:
(seems like the default is 6, but the wiki says 5)

Reply:
We want the length at 5

Reply:
Made it 5